### PR TITLE
LRQA-70190 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-70190

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase [hunk 1]
```diff
@@ -1318,35 +1318,41 @@ definition {
...
-		Accounts.addBusinessAccount(
-			accountDescription = "New Active Account Selection",
-			accountName = "New Selection for Test");
+			Accounts.addBusinessAccount(
+				accountDescription = "New Active Account Selection",
+				accountName = "New Selection for Test");
+		}
 
-		Navigator.gotoPage(pageName = "2B Account Page");
+		task ("Select new account to be main active account, denoted by a checkmark, and assert that previous account is no longer selected") {
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase:1343]: @epark18 These can be separate tasks. Can you create another refactoring ticket to convert 2BAccount test tasks to be BDD? Thank you!

**Review by patriciajeanperez — COMMENTED** [review]
@epark18 looks really good! I like how we were able to simplify the macro to only do what it's supposed to do. I will forward this but can we create a ticket for the refactoring of the tasks? Thanks!

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**